client/jetpack-cloud/sections/overview/primary/intro-cards/index.tsx

```diff
@@ -0,0 +1,88 @@
+import { useTranslate } from 'i18n-calypso';
+import DotPager from 'calypso/components/dot-pager';
+
+import './style.scss';
+
+const Card1 = () => {
+	const translate = useTranslate();
+	return (
+		<>
+			<h1>{ translate( '👋 Welcome to Jetpack Manage' ) }</h1>
```

Comment:
Something I'm not sure about is how the emoji is translated. I've noticed a different approach elsewhere, for example in `client/my-sites/checkout/checkout-thank-upi/licensing-thank-you-auto-activation-completed.tsx` I see the following:

```
translate( `Your %(productName)s is active! %(celebrationEmoji)s`, {
					args: {
						productName: hasProductInfo ? ( productName as string ) : 'subscription',
						celebrationEmoji: String.fromCodePoint( 0x1f389 ) /* Celebration emoji 🎉 */,
					},
			  } )
```

I haven't seen any examples of emoji passed straight into translate functions in the wp-calypso codebase.

Searching through the 'i18n-calypso' package code I haven't yet found anything clear, though the readme does say that i18n-calypso requires strings only to passed into the translate function, so the examples converting the emoji to a string would make sense in this respect.

Reply:
If this is a better approach, I've been looking at this more for another tour which has an emoji in the title. At the moment the option I came up with in my case is the following, so similar should apply in your case:
```
title: String(
	translate( '%(celebrationEmoji)s Your new site is here!', {
		args: {
			celebrationEmoji: String.fromCodePoint( 0x1f389 ) /* Celebration emoji 🎉 */,
		},
	} )
),
```

Reply:
An emoji is just a character like any other; is there any reason to believe it should be "translated" differently? I did find at least one example:

https://github.com/Automattic/wp-calypso/blob/1ee1ee59986a0bcff326dac61534127eefbf7f7f/client/my-sites/marketing/business-tools/index.tsx#L130

Reply:
And it looks like it was translated successfully:

![image](https://github.com/Automattic/wp-calypso/assets/32492176/60684310-d58b-4a9d-809e-359d7a2538a6)

Reply:
Oh perfect. Ignore my suggestions then :) I couldn't find any documentation to confirm otherwise nor spot an example like that one.